SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/SpatialGDKEditorSchemaGeneratorTest.cpp

```diff
@@ -1051,4 +1051,155 @@ SCHEMA_GENERATOR_TEST(GIVEN_no_schema_exists_WHEN_generating_schema_for_rpc_endp
 	return true;
 }
 
+SCHEMA_GENERATOR_TEST(GIVEN_actor_class_WHEN_generating_schema_THEN_expected_component_set_filled)
+{
+	SchemaTestFixture Fixture;
+
+	TSet<UClass*> Classes = { ASpatialTypeActor::StaticClass(), USchemaGenObjectStubHandOver::StaticClass(),
```

Comment:
This seems like 3 test cases. In order to simplify any single test, as this one is complicated (necessarily so to an extent), could the logic of the test be extracted into a separate function and we split this into three tests where we pass each class in? Or is there a benefit to having it all in one test?

Reply:
I could make one actor with both handover and owner only properties. Test would remain the same. It just adds a little variation that actually helped me catch a few bug. I could actually use the AllClasses array, test would remain the same.